.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -15,7 +15,7 @@ AGENTS_LABELS = [
 
 
 def simulationContainerTest(String version, String build_type, List extra_cmake_args = []) {
-    stage("Simulation Ubuntu ${version} clang-8 ${build_type}, extra_cmake_args: ${extra_cmake_args}") {
+    stage("Simulation Ubuntu ${version} clang-10 ${build_type}, extra_cmake_args: ${extra_cmake_args}") {
```

Comment:
I'd like to suggest that just using `clang` instead of `clang-version` on the stage name (here and line 113 - 118, 125 - 128). This will minimize the change for each clang update.